RocketChat/Rocket.Chat
# [FIX] Omnichannel call provider option

## Proposed changes (including videos or screenshots)


Jira Issue: [VC-38]



## Issue(s)


## Steps to test or reproduce

- Install some conf app
- Go to Administration > Omnichannel
- Check if there's the Default Provider and None options

## Further comments



[VC-38]: https://rocketchat.atlassian.net/browse/VC-38?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
